src/input_cp2k_dft.F

```diff
@@ -9721,4 +9729,143 @@ SUBROUTINE create_cdft_opt_section(section)
 
    END SUBROUTINE create_cdft_opt_section
 
+   SUBROUTINE create_dft_plus_u_section(section)
+      TYPE(section_type), POINTER                        :: section
+
+      TYPE(keyword_type), POINTER                        :: keyword
+
+      CPASSERT(.NOT. ASSOCIATED(section))
+
+      CALL section_create(section, __LOCATION__, &
+                          name="DFT_PLUS_U", &
+                          description="Define the parameters for the DFT+U correction implemented in the pwdft method", &
+                          n_keywords=4, &
+                          repeats=.FALSE.)
+      NULLIFY (keyword)
+
+      CALL keyword_create(keyword, __LOCATION__, &
+                          name="_SECTION_PARAMETERS_", &
+                          description="Controls the activation of the DFT+U section", &
+                          usage="&DFT_PLUS_U ON", &
```

Comment:
How does this new `DFT_PLUS_U` section relate to the existing [`PLUS_U_METHOD`](https://manual.cp2k.org/trunk/CP2K_INPUT/FORCE_EVAL/DFT.html#CP2K_INPUT.FORCE_EVAL.DFT.PLUS_U_METHOD)? Could you maybe just extend the existing keyword?